incidenter/height.py

```diff
@@ -0,0 +1,41 @@
+from os import getenv
+
+from autokitteh.height import height_client
+
+
+height = height_client("height")
+
+
+LIST_ID = getenv("HEIGHT_LIST_ID")
```

Comment:
Ensure LIST_ID is never `None`
```suggestion
LIST_ID = os.getenv("HEIGHT_LIST_ID", "")
```

Reply:
why does it matter? None or "" in json serializing further ahead will yield an error anyway.

Reply:
added a check